### ISSUE TITLE
Some MAC addresses trigger rule 210

### ISSUE THREAD

**Issue by cr1xu5** [warpnet/salt-lint#202]
First of all, thank you for this tool!

**Describe the bug**

Some MAC addresses trigger rule 210.
This is the case then the two last characters are integers.

**To Reproduce**
```sls
valid: 00:e5:e5:aa:60:69
valid2: 09:41:a2:48:d1:6f
valid3: 4c:f2:d3:1b:2e:0d
invalid: 4c:f2:d3:1b:2e:05
```

**Expected behavior**
All of the above are valid (or all MAC addresses should be encapsulated).

```sh
pip3 show salt-lint
Name: salt-lint
Version: 0.4.2
```

**Comment by jbouter** [comment]
Hi @cr1xu5. Thanks for reporting this issue! We've fixed this in #206, which will be included in the next release.
